mlprojects/sklearn-diabetes/environment.yaml

```diff
@@ -0,0 +1,12 @@
+name: mlflow-diabetes
+channels:
+  - defaults
+dependencies:
+  - numpy>=1.14.3
+  - pandas>=1.0.0
+  - scikit-learn=0.19.1
```

Comment:
does scikit-learn need to be pinned so low?

Reply:
got rid of them. Thanks for that!